Flabbergasted-Flamingos/Bangazon-admin-app
# Jcemployeedetailedlink

# Description
Added the link to the employee name of the list view. The link connects the employee name to the detailed view of the employee

## Number of Fixes
1

## Problem to Solve
Read the description 

## Steps to Test Solution

1. git fetch --all
2. git checkout JCemployeedetailedlink
3. go to http://localhost:8000/admin_app/employees/
4. Check when you click employee name if it goes to the detailed view for that employee
